i18n/de.toml

```diff
@@ -61,28 +61,28 @@ savingsTotalEnergy = "{total} kWh geladen"
 tabTitle = "Live-Community"
 power = "Ladeleistung"
 powerSub1 = "{activeClients} von {totalClients} Nutzern"
-powerSub2 = "laden mit evcc"
+powerSub2 = "laden..."
 greenShare = "Sonnenanteil"
-greenShareSub1 = "der Leistung kommt"
+greenShareSub1 = "Leistung kommt von"
 greenShareSub2 = "aus PV & Hausakku"
-greenEnergy = "Sonnenenergie"
+greenEnergy = "Sonne"
 greenEnergySub1 = "über evcc geladen"
 greenEnergySub2 = "seit Oktober 2022"
 
 [footer.telemetry]
 optIn = "Ich möchte meine Ladedaten mit der Community teilen."
-optInMoreDetails = "Mehr Details gibts {0}."
+optInMoreDetails = "Mehr Details {0}."
 optInMoreDetailsLink = "hier"
 optInSponsorship = "Sponsoring erforderlich."
 
 [footer.sponsor]
 titleSponsor = "Du bist Unterstützer"
 titleNoSponsor = "Unterstütze uns"
-thanks = "Danke für dein Sponsoring, {sponsor}! Das hilft uns bei der Weiterentwicklung."
+thanks = "Vielen Dank, {Sponsor}! Dein Beitrag hilft, evcc weiterzuentwickeln."
```

Comment:
Seit neuestem ja

    weiter zu entwickeln

Reply:
Dann bitte in Weblate anpassen.